src/components/ciConfig/CIConfig.tsx

```diff
@@ -229,7 +229,7 @@ function Form({ dockerRegistries, sourceConfig, ciConfig, reload, appId }) {
     };
 
     const containerRegistryOption = (props) => {
-        props.selectProps.styles.option = getCustomOptionSelectionStyle({ direction: 'none' });
+        props.selectProps.styles.option = getCustomOptionSelectionStyle();
```

Comment:
Can we remove this 'getCustomOptionSelectionStyle()' as this will anyway be the default option

Reply:
we are using getCustomOptionSelectionStyle() css directly when someone creates a custom option

Reply:
As per our discussion we are not removing getCustomOptionSelectionStyle